packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -50,8 +50,8 @@
 .ds-c-alert__a11y-label {
   margin-right: 0.5rem;
 
-  // !important needed to overwrite screen reader utility styles
   @media (-ms-high-contrast: active), (forced-colors: active) {
+    /* stylelint-disable declaration-no-important -- needed to overwrite screen reader utility styles */
```

Comment:
Inlined the comment as part of the [lint exception description](https://stylelint.io/user-guide/usage/options/#reportdescriptionlessdisables).

For a11y reasons, we have a visually hidden screen reader label to let users on AT know what kind of alert they're reading (since our alert icons aren't visible). HCM users also need a visual queue, so we overwrite the CSS rules when HCM is enabled. This cannot be refactored in a way to remove `!important` as that specificity is needed for overwriting the SR utility class.